### DISCUSSION TITLE
0x8a15000f : Data required by the source is missing

### DISCUSSION THREAD

**Discussion by LeGrosWinnie** [Devolutions/UniGetUI#1087] [Q&A]
Hello,

When using the standard command to install wingetUI I get the "classic" error about "network" etc.

0x8a15000f : Data required by the source is missing

Any information about that ?

Edit: WIndows 11 Preview, Winget automatically installed v1.5.1361-preview
I think this version is totally bugged...
A lot of command failled like :

winget source update
Mise à jour en cours de toutes les sources... Merci de patienter.
Mise à jour de la source : msstore...
Terminé
**Mise à jour de la source : winget...
Annulé**

So in English : Update source: winget cancelled...

Thanks.

**Comment by marticliment** [comment]
Hello @LeGrosWinnie,

what is your user directory path? Non-ascii characters sometimes give errors...

**Reply by LeGrosWinnie** [reply]
Hello,
C:\Users\CodeKiller
If there are logs or anything else somewhere I can look.
Thanks.

**Reply by marticliment** [reply]
Can you please try the reset sources command from wingetui preferences, under Winget Package Manager Preferences?

**Comment by LeGrosWinnie** [comment]
Hello again,

I installed the 2.0 beta, just for testing purposes, before installing it I
saw packages were found for all except winget so I did what you asked. In
the terminal window I had 3 lines "resource not found" or something like
that (when a file/folder does not exist) but still exit normally.
As I reinstalled it it is still running to get sources.

Note that he correctly finds everything already installed. The only issue
seems to get Winget source, which is kind of ironic for... Winget... :-)

I'll get you in touch when I have news.


Le jeu. 8 juin 2023 à 17:42, Martí Climent ***@***.***> a
écrit :
